### DISCUSSION TITLE
明天黑猴上线之后肯定会有这个东西

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#3150] [其他]
<img width="540" alt="截屏2024-08-19 21 06 28" src="https://github.com/user-attachments/assets/4601cc32-a597-4ff9-bb27-ec30f2b9f49a">
你铁文案的含金量还在增加

**Comment by wilkerCoder** [comment]
还没上线的东西，还是不要发比较好<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by cyq20091029** [reply]
应该没啥问题吧，b站上有不少

**Comment by warning-Kirov-reporting** [comment]
流量大的东西都会有的
